graph/builder.go

```diff
@@ -701,28 +697,35 @@ func (b *Builder) handleNetworkUpdate(update *routingMsg) {
 			log.Errorf("process network updates got: %v", err)
 		}
 
-		return
+		return err
 	}
 
-	// Otherwise, we'll send off a new notification for the newly accepted
-	// update, if any.
-	topChange := &TopologyChange{}
-	err = addToTopologyChange(b.cfg.Graph, topChange, update.msg)
-	if err != nil {
-		log.Errorf("unable to update topology change notification: %v",
-			err)
-		return
-	}
+	b.wg.Add(1)
+	go func() {
+		defer b.wg.Done()
```

Comment:
Any reason we want this to be async?

Reply:
just keeping the logic as close to today's as possible here since today this part is async. Perhaps doing the topology notifications takes a while, for example. Callers of AddNode/Channel should only be blocked on the DB writes I think

Reply:
N/A now as i've reverted to some of the previous logic (but this part still remains async there)